projects/gigya/src/auth/services/user-authentication/gigya-user-authentication-token.service.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { UserToken, OccEndpointsService, AuthConfig } from '@spartacus/core';
+import { catchError } from 'rxjs/operators';
+
+@Injectable()
+export class GigyaUserAuthenticationTokenService {
+  constructor(
+    protected http: HttpClient,
+    protected config: AuthConfig,
+    protected occEndpointsService?: OccEndpointsService
```

Comment:
Make it required contructor parameter
```suggestion
    protected occEndpointsService: OccEndpointsService
```